Adds Dracthyr race, Devastation and Preservation spells

joshpc opened the pull request:
# What this does
Adds the library of new abilities for Evoker added in the 10.0 pre-patch.

_Disclaimer:_ I mainly used Wowhead for the CD times and caught a few mistakes already. I believe I caught them all but a second set of eyes would be good.

## Testing Done
- Copied it directly into my version of GladiusEX and seems to be mostly okay. The only issue I've noticed is that the Dracthyr icon (under racials) does not show up.

joshpc committed:
Adds evoker cooldowns

joshpc committed:
Fixes incorrect spellIds for Time Spiral and Emerald Communion

joshpc committed:
Adds racials and optional charges / cooldown reductions

vendethiel commented on cooldowns_evoker.lua:
```diff
@@ -0,0 +1,390 @@
+LCT_SpellData[357208] = {
+  class = "EVOKER",
+  cooldown = 30,
+  -- TODO: Fire Breath's CD is reduced when consuming essence bursts when
```
I can't wait having to try to track that..:-)

vendethiel commented on cooldowns_evoker.lua:
```diff
@@ -0,0 +1,390 @@
+
+-- Azure Strike
+LCT_SpellData[362969] = {
+  class = "EVOKER"
```
Why does this not have a cooldown?

joshpc replied:
I blindly put all the spells (even those that are spammable.) Should I take them out?

vendethiel replied:
The only time I tend to do that is when that spell has special properties (for example, fireball reducing combustion cooldown, or spells reducing hammer of justice cooldown). And I add a `hidden = true` property so they’re not displayed in the options’ cooldown list.

vendethiel commented on cooldowns_evoker.lua:
```diff
@@ -0,0 +1,390 @@
+
+-- Return
+LCT_SpellData[361227] = {
+  class = "EVOKER"
```
No cooldown either?

joshpc committed:
Removes spells with no CDs

They have no known interactions with existing spells and do not need to be listed
